plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/plugin.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?eclipse version="3.4"?>
+<plugin>
+   <extension point="org.eclipse.ui.newWizards">
+      <wizard
+            id="com.google.cloud.tools.eclipse.appengine.newproject.MavenAppEngineStandard"
+            name="%mavenwizard.name"
+            class="com.google.cloud.tools.eclipse.appengine.newproject.maven.MavenArchetypeProjectWizard"
+            icon="platform:/plugin/com.google.cloud.tools.eclipse.appengine.newproject/icons/app_engine_standard_16x16.png"
+            project="true"
+            category="com.google.cloud.tools.eclipse.appengine.wizards">
+         <description>%mavenwizard.description</description>
+         <selection class="org.eclipse.core.resources.IResource"/>
+      </wizard>
+   </extension>
+   
+   <extension
+         point="org.eclipse.m2e.core.lifecycleMappingMetadataSource">
+   </extension>
+   <extension
+         point="org.eclipse.wst.xml.core.catalogContributions">
+      <catalogContribution>
+         <uri
+               name="http://appengine.google.com/ns/1.0"
+               uri="schema/appengine-web.xsd">
+         </uri>
+      </catalogContribution>
+   </extension>
+   <extension
+         id="id1"
+         point="org.eclipse.wst.validation.exclude">
+      <validator
+            id="org.eclipse.wst.xml.core.xml">
+         <exclude>
+            <rules>
+               <file
+                     name="nbactions.xml"
```

Comment:
Can we just exclude appengine-web.xml as well rather than adding the schema? The schema really isn't necessary. I suppose we might at some point validate it at milestone 3 (the quick fixes and code inspections) but if so we'll want to pull the schema from gcloud rather than bundling it ourselves.